_posts/en/newsletters/2019-03-26-newsletter.md

```diff
@@ -0,0 +1,263 @@
+---
+title: 'Bitcoin Optech Newsletter #39'
+permalink: /en/newsletters/2019/03/26/
+name: 2019-03-26-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a proposal to encrypt P2P communication
+and describes Lightning Loop, a tool and service for withdrawing
+bitcoins from an LN channel to an onchain transaction.  Also included
+are links to resources about bech32 adoption, summaries of popular
+questions and answers from Bitcoin StackExchange, and a list of notable
+code changes in popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Help test Bitcoin Core 0.18.0 RC2:** The second Release Candidate
+  (RC) for the next major version of Bitcoin Core has been [released][0.18.0].
+  Testing is still needed by organizations and experienced users who
+  plan to run the new version of Bitcoin Core in production.  Use [this
+  issue][Bitcoin Core #15555] for reporting feedback.
```

Comment:
Please see this comment https://github.com/bitcoin/bitcoin/issues/15555#issuecomment-473919220:

> I'd advise against reporting issues here. They should go in their own report. Otherwise it is hard to follow-up on reports when everything is in the same thread.

Reply:
Thanks @practicalswift! I think it still makes sense to point testers to a single location to 'leave feedback'. If they need to open an new issue for an unreported problem, that issue instructs them to do so.